pkg/controller/kfservice/resources/credentials/service_account_credentials.go

```diff
@@ -61,7 +60,7 @@ func NewCredentialBulder(client client.Client, config *v1.ConfigMap) *Credential
 }
 
 func (c *CredentialBuilder) CreateSecretVolumeAndEnv(namespace string, serviceAccountName string,
-	configuration *knservingv1alpha1.Configuration) error {
+	container *v1.Container, volumes *[]v1.Volume) error {
```

Comment:
+1

Reply:
Actually, better to pass in the template spec? It's ducktyped so it's generic to k8s v1 right?

Reply:
Not sure how to do that - can you do that in a later change?